src/signals/incident/components/form/RadioInputGroup/index.test.js

```diff
@@ -60,6 +54,32 @@ describe('Form component <RadioInput />', () => {
       expect(wrapper).toMatchSnapshot();
     });
 
+    it('should render info from current selection correctly', () => {
```

Comment:
Is this not out of scope here? Maybe an separate chore PR?

Reply:
I added it because of a coverage error was triggered

Reply:
Removed this test